assets/heliumjk/css/main.css

```diff
@@ -1,4 +1,4 @@
 @media (max-width: 480px){.navbar-brand img{max-width:120px}.container .navbar-toggler{padding:15px 15px!important;}}
-@media (max-width:767px){.container{margin-left:0; margin-right:0;width:100%;}.container .navbar-toggler{position: absolute;right:0px; padding:25px 15px; margin: 10px;}}.mb-25{margin-bottom:25px;}.div-catalog-img{overflow:hidden;max-height:150px;}.div-blog-img{overflow:hidden;max-height:400px;}.div-img{width:100%;height:auto}.text-version{color:#fff;background-color:#555;padding:1px 6px;border-radius:5px;}
+@media (max-width:767px){.container{margin-left:0; margin-right:0;width:100%;}.container .navbar-toggler{position: absolute;right:0px; padding:25px 15px; margin: 10px;}}.mb-25{margin-bottom:25px;}.div-catalog-img{overflow:hidden;max-height:150px;}.div-blog-img{height:300px;display:flex;justify-content:center;align-items:center;}.div-img{width:auto;height:auto;max-width:100%;max-height:100%;display:block;}.text-version{color:#fff;background-color:#555;padding:1px 6px;border-radius:5px;}[role="button"]:focus{outline:none;}
```

Comment:
based on other input, seems like we may need to go even smaller on the images, or reformat the all-posts display

Reply:
Made another version. Please take a check.